sale_report_product_image/views/sale_product_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_order_image_form_inherit" model="ir.ui.view">
+            <field name="name">sale.order.form.sale.image</field>
+            <field name="model">sale.order</field>
+            <field name="inherit_id" ref="sale.view_order_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//page[@string='Order Lines']/field[@name='order_line']/form[@string='Sales Order Lines']/group/group/field[@name='product_id']" position="before">
+                    <field name="image_small" widget="image"/>
```

Comment:
AFAICT this adds the image in the embedded tree for the SO lines.
Are you sure it works? Because [this widget](https://github.com/OCA/web/tree/8.0/web_tree_image) used to be necessary for that.